Dream-Home-PEABOT/FEE_DreamHome
# Home component testing

### What does this PR do?
- It adds unit testing for the Home component

### Where should the reviewer start?
`src/component/Home/Home.src`

### How should this be manually tested?
- run `npm `test on the CLI, then press `p` then type the component you want to see being evaluated
### Any background context you want to provide?
n/a
### What are the relevant tickets?
closes #78 
### Questions:

### Screenshots (if appropriate)
<img width="683" alt="Screen Shot 2021-01-12 at 1 51 03 PM" src="https://user-images.githubusercontent.com/56229864/104373947-60e49c80-54de-11eb-80a2-d5a6ec9413d3.png">
